### ISSUE TITLE
Generating cache-able targets as default

### ISSUE THREAD

**Issue by avadhanij** [lucasvieirasilva/nx-plugins#261]
- [ ] I'd be willing to implement this feature

## Description

While playing around with this plugin, I noticed that the default `project.json` that's generated doesn't have `"cache": true` set on any of the targets.

Was curious if there was a reason for this or was just something left up to the end user to do? I imagine `build`, `test` and `lint` would all normally be cache-able.

## Motivation

Just a nit enhancement

## Suggested Implementation

Make the default generated targets `build`, `test` and `lint` cacheable.

**Comment by lucasvieirasilva** [comment]
Hi @avadhanij,

On Nx older versions, when you created a workspace, the `lint`, `test`, and `build` were cache-ble as default, and those configurations were global.

I just created a Nx workspace from scratch, and the config is not there by default, I don't know the reason, but I found this doc https://nx.dev/features/cache-task-results#define-cacheable-tasks that shows how to configure cache-ble tasks globally.

but I guess I could add a `cache: true` to those `project.json` from the project generator.
